dask/dataframe/core.py

```diff
@@ -3749,14 +3750,7 @@ def _validate_axis(cls, axis=0):
     def drop(self, labels=None, axis=0, columns=None, errors="raise"):
         axis = self._validate_axis(axis)
         if (axis == 1) or (columns is not None):
-            return self.map_partitions(
-                M.drop,
-                labels=labels,
-                axis=axis,
-                columns=columns,
-                errors=errors,
-                enforce_metadata=False,
-            )
+            return self.map_partitions(drop_by_shallow_copy, columns or labels)
```

Comment:
I think that we should pass down the `errors=` keyword into the inplace drop call within this function.